lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/IssueCredentialHandler.java

```diff
@@ -155,6 +160,12 @@ public APIGatewayProxyResponseEvent handleRequest(
             LOGGER.info("Validating authorization token...");
             var accessToken = validateInputHeaderBearerToken(input.getHeaders());
             var sessionItem = this.sessionService.getSessionByAccessToken(accessToken);
+
+            if (sessionItem == null || sessionItem.getSessionId() == null) {
+                LOGGER.error("Session is not found");
```

Comment:
Can you remove the log line from here.

As the catch block will log this also.

Reply:
Fixed in all 3 CRIs